### PR TITLE
RAC-223: security login test

### PR DESCRIPTION
**Pull request by diontje**
RAC-223: security login test
* updated security login tests from OnRack/Autotest to test against RackHD
* tests login with current RackHD roles, User, Administrator, and Operator

### CODE REVIEW

**Comment by diontje** [comment]
test this please

**Comment by diontje** [comment]
test this please

**Comment by hohene** [comment]
test this please

File: test/dellemc-test [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/cuthbt1/sandbox/dellemc-test
```
Comment (hohene) [inline test/dellemc-test]: Do we want this file here?  looks wrong.

File: test/tests/security/test_api_security_login.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+# Helper functions
+def createUsername():
+    return exrex.getone('[a-zA-Z]{1}[a-zA-Z0-9._\-]{1,}')
+
+
+@attr(regression=True, smoke=True, security=True)
+class TestCase01_RackHDAPI_account_creation(fit_common.unittest.TestCase):
+
+    def setUp(self):
+        logs.info("Running test: %s", self._testMethodName)
```
Comment (hohene) [inline test/tests/security/test_api_security_login.py]: remove this log.info, test info is printed by default.

File: test/tests/security/test_api_security_login.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        self.assertIsNotNone(admin)
+        self.assertIsNotNone(readonly)
+        self.assertIsNotNone(operator)
+
+
+@attr(regression=True, smoke=True, security=True)
+class TestCase02_RedfishAPI_account_creation(fit_common.unittest.TestCase):
+
+    def setUp(self):
+        logs.info("Running test: %s", self._testMethodName)
```
Comment (hohene) [inline test/tests/security/test_api_security_login.py]: remove this log.info, test info is printed by default.

File: test/dellemc-test (hunk 1 above)
Comment (diontje) [inline test/dellemc-test]: oops. symbolic link to the config repo, removed

File: test/tests/security/test_api_security_login.py (hunk 2 above)
Comment (diontje) [inline test/tests/security/test_api_security_login.py]: done

File: test/tests/security/test_api_security_login.py (hunk 3 above)
Comment (diontje) [inline test/tests/security/test_api_security_login.py]: done

**Pull request merged**